src/SignUp/SignUP.module.css

```diff
@@ -0,0 +1,219 @@
+  font-weight: 700;
+}
+.signuptxt:hover {
+  background-color: #d3cfd0;
+}
+
+.logo {
+  color: #e60023;
+  cursor: pointer;
+  font-weight: 700;
+  margin-top: -40px;
+}
+
+.headertxt {
+  font-size: 200%;
+  font-weight: 600;
+  margin-top: -15px;
+}
+
+.headertxt2 {
+  position: relative;
+  bottom: 35px;
+  font-size: 15px;
+  font-weight: 400;
+  color: rgb(131, 126, 126);
+}
+
+.logintxt {
+  color: #fff;
+  background-color: #e60023;
+  padding: 8px 12px 8px 12px;
+  border-radius: 30px;
+  font-weight: 700;
+}
+.logintxt:hover {
+  background-color: rgb(206, 46, 46);
+}
+
+.cancel {
+  font-weight: 1000;
+  font-size: 1000px;
+}
+
+.input {
+  margin-top: -45px;
+  border-bottom-width: 0px;
+  outline: none;
+  width: 52%;
+  padding: 17px;
+  border-radius: 15px;
+  border: 2px solid rgb(212, 208, 203);
+}
+
+.input:hover {
+  border: 2px solid #e60023;
+}
+.input:focus {
+  border: 2px solid rgb(212, 208, 203);
+  outline-color: #68b9e7;
+  outline-style: solid;
+  outline-width: 4px;
+}
+
+.inputPass {
+  margin-top: -50px;
+  outline: none;
+  width: 52%;
+  padding: 17px;
+  border-radius: 15px;
+  border: 2px solid rgb(206, 203, 199);
+}
+
+.inputPass:hover {
+  border: 2px solid #e60023;
+}
+.inputPass:focus {
+  border: 2px solid rgb(242, 243, 240);
+  outline-color: #68b9e7;
+  outline-style: solid;
+  outline-width: 4px;
+}
+
+.inputAge {
+  position: relative;
+  bottom: 35px;
+  margin-top: -18px;
+  outline: none;
+  width: 52%;
+  padding: 17px;
+  border-radius: 15px;
+  border: 2px solid rgb(206, 203, 199);
+}
+
+.inputAge:hover {
+  border: 2px solid #e60023;
+}
+.inputAge:focus {
+  border: 2px solid rgb(242, 243, 240);
+  outline-color: #68b9e7;
+  outline-style: solid;
+  outline-width: 4px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.forgetxt:hover {
+  text-decoration: 1.4px underline;
+}
+
+.logintxt {
+  margin-top: -45px;
+  box-sizing: border-box;
+  padding: 8px;
+  padding-left: 24%;
+  padding-right: 24%;
+  background-color: #e60023;
+}
+
+.logintxt:focus {
+  outline-color: #68b9e7;
+  outline-style: solid;
+  outline-width: 4px;
+}
+
+.orTxt {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.facebook {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 10px;
+  box-sizing: border-box;
+  padding: 8px;
+  padding-left: 6%;
+  padding-right: 6%;
+  background-color: #4285f4;
+  border-radius: 30px;
+  font-size: 90%;
+  font-weight: 700;
+  color: #fff;
+}
+
+.google {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 10px;
+  box-sizing: border-box;
+  padding: 8px;
+  padding-left: 6%;
+  padding-right: 6%;
+  background-color: #fff;
+  border-radius: 30px;
+  font-size: 90%;
+  font-weight: 700;
+  border: 1px solid rgb(206, 203, 199);
+}
+
+.Txt {
```

Comment:
Do practice to keep class name or variable name in same manner, either in camel case or snake case etc.. (**.Txt** ~ **.txt**)

Reply:
And add meaning full names (Don't add like **.Txt**)